app/controllers/signup_controller.rb

```diff
@@ -21,7 +21,8 @@ class SignupController < ApplicationController
 
     if @user.save
       session.delete(:error) if session[:error]
-      redirect to('/dashboard')
+      session[:email] = @user.email
```

Comment:
Dunno what the goal is here, but it's generally bad practice to pass user information like this in plain sight

Reply:
I have addressed this